cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-scaling.adoc

```diff
@@ -0,0 +1,259 @@
+
+.Prerequisites
+
+* Create a cluster with the xref:../../cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview.adoc[OSToy microservice deployed].
+* Download the link:https://www.rosaworkshop.io/ostoy/yaml/ostoy-microservice-deployment.yaml[ostoy-microservice-deployment.yaml].
+
+.Procedure 
+
+. In the OSToy app, click the *Networking* tab in the left menu.
+. In the "Intra-cluster Communication" section, there is a box that randomly changes colors. Inside the box, you see the microservice's pod name. There is only one box shown because there is only one microservice pod. 
++
+image::deploy-scale-network.png[HPA Menu]
++ 
+Now, scale up the microservice pods and observe the number of boxes change.
+
+. Run the following command to confirm that there is only one pod running on the microservice:
++
+[source,terminal,highlight='4']
+----
+$ oc get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                  READY     STATUS    RESTARTS   AGE
+ostoy-frontend-679cb85695-5cn7x       1/1       Running   0          1h
+ostoy-microservice-86b4c6f559-p594d   1/1       Running   0          1h
+----
+
+. Now, scale the application by using the CLI or using the Web UI:
++
+** Run the following command increases the number of pods from `1` to `3`:
++
+[source,terminal]
+----
+$ oc scale deployment ostoy-microservice --replicas=3
+----
++
+** In the OSToy application, from the left menu, click *Workloads > Deployments > ostoy-microservice*.  On the left side of the page, you see a blue circle with a "1 Pod" label in the middle. Selecting the arrows next to this scales the number of pods. Select the up arrow to `3`.
++
+image::deploy-scale-uiscale.png[UI Scale]
+
+.Verification
+
+Check your pod counts with the following methods:
+
+* In the web UI, select *Workloads > Deployments > ostoy-microservice*.
++
+image::deploy-scale-verify-workload.png[Verify the workload pods]
+
+* From the CLI, run the follow command to confirm that you are using three pods:
++
+[source,terminal]
+----
+$ oc get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                  READY     STATUS    RESTARTS   AGE
+ostoy-frontend-679cb85695-5cn7x       1/1       Running   0          1h
+ostoy-microservice-86b4c6f559-p594d   3/3       Running   0          1h
+----
+
+* You can also confirm that there are three pods being used by selecting **Networking** in the left menu of the OSToy app. There should be three colored boxes for the three pods.
++
+image::deploy-scale-colorspods.png[UI Scale]
+
+=== Pod Autoscaling
+
+{product-title} offers a link:https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-autoscaling.html[Horizontal Pod Autoscaler] (HPA). The horizontal pod autoscaler (HPA) uses metrics to decide if increase or decrease the scale your cluster's worker pods.
```

Comment:
The second sentence needs updating.  I think something to the effect of "The HPA uses metrics to decide whether to increase of decrease the number of pods."